pyphare/pyphare_tests/pharein/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required (VERSION 3.3)
+cmake_minimum_required (VERSION 3.20.1)
```

Comment:
TC was showing the following text

```
CMake Deprecation Warning at pyphare/pyphare_tests/test_pharesee/CMakeLists.txt:1 (cmake_minimum_required):
```

so no point in not updating these three files to be consistent with the rest of the project